code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -315,6 +324,11 @@ emp_act
 			var/shield_check = check_shields(throw_damage, O, thrower, zone, "[O]")
 			if(shield_check == PROJECTILE_FORCE_MISS)
 				zone = null
+			else if(shield_check == PROJECTILE_REFLECT)
+				//var/obj/item/weapon/B = O
+				var/obj/item/weapon/ball/B = O

```

Comment:
It would probably be best if reflected() was made a atom/movable proc instead of being specific to balls, so that PROJECTILE_REFLECT can be used on those and it won't break if for some reason it gets fed that with something that's not a ball.